pkg/fanal/analyzer/pkg/apk/apk.go

```diff
@@ -206,3 +214,23 @@ func (a alpinePkgAnalyzer) Type() analyzer.Type {
 func (a alpinePkgAnalyzer) Version() int {
 	return analyzerVersion
 }
+
+// decodeChecksumLine decodes checksum line
+func decodeChecksumLine(line string) digest.Digest {
+	// https://wiki.alpinelinux.org/wiki/Apk_spec#Package_Checksum_Field
+	// https://stackoverflow.com/a/71712569
+	alg := digest.MD5
+	d := line[2:]
```

Comment:
It is better to check the length of `line` just in case. It will cause panic if the length is less than two.

Reply:
Done